doc/_extensions/zephyr/domain/templates/board-catalog.html

```diff
@@ -17,7 +17,6 @@
     <div class="select-container">
       <select id="arch">
         <option value="" disabled selected>Select an architecture</option>
-        <option value="nios2">Altera Nios II</option>
```

Comment:
opened https://github.com/zephyrproject-rtos/zephyr/pull/90413 so that we don't have to do this ever again :)